<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="BotonLeeme">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Leeme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonAcercaDe">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Acerca de...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonConfig">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Configuración</string>
    </property>
   </widget>
   <widget class="QFrame" name="CajaConfig">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="TextProductor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Productor</string>
     </property>
    </widget>
    <widget class="QLabel" name="TextConsumidor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Consumidor</string>
     </property>
    </widget>
    <widget class="QDialogButtonBox" name="BotonOK">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>166</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="DatoProd">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="DatoCons">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="TextoAcercaDe">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>80</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Universidad: La Salle Arequipa&lt;br/&gt;Curso: Sistemas Operativos&lt;br/&gt;Profesor: Richart Escobedo&lt;br/&gt;Alumno: Aaron J. M. Choy Romero&lt;br/&gt;Semestre : 5&lt;/span&gt;&lt;span style=&quot; font-size:14pt; vertical-align:super;&quot;&gt;to&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;&lt;br/&gt;Año académico: 2021-1&lt;br/&gt;Fecha: 13/06/2021&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="TextoLeeme">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>311</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Productor-Consumidor&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Es un problema clasico sobre&lt;br /&gt;hilos. Es un proceso donde dos &lt;br /&gt;procesos (el productor y el &lt;br /&gt;consumidor) comparten un buffer&lt;br /&gt;que en cierto punto se puede llenar&lt;br /&gt;o puede quedarse vacio la mayoria &lt;br /&gt;del tiempo.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
